accessibility.md

```diff
@@ -1,19 +1,26 @@
 ---
-layout: default
+layout: blue_page
 permalink: /accessibility/
 title: Accessibility Statement
 ---
 
-<div class="popout">
-  <section>
-      <h1>Accessibility Statement</h1>
-  </section>
-</div>
+> You may also be interested in the [**venue accessibility statement**](http://www.focusterra.ethz.ch/en/your-visit.html).
 
-<section markdown="1">
+RustFest is an inclusive conference and as such strives to be accessible to everyone. This includes:
 
-* The venue is wheelchair accessible, the toilet too!
-* There will be a a quiet room on Saturday, wheelchair accessible.
-* Please do not wear scents.
+* Offering significantly discounted tickets for concession holders.
+* Ensuring our venues are accessible by wheelchair.
+* Make sure that maps and orientation material is available.
+* Providing clear signage to, from and through the building
+* Providing quiet spaces for those with sensory issues.
+* Providing accessibility for conference sessions.
+* Providing complimentary tickets for assistants for those with disabilities.
+* Access to non-gendered toilets.
```

Comment:
(sorry for the duplicated comment - I somehow managed to accidentally delete my previous one)

In general, the statement looks (very) good to me. However, at this point we (sadly) can not yet guarantee non-gendered toilets - the only exception being the special toilets for people with physical disabilities. I don't think this counts, does it?

I still have to talk to people at ETH about this - this might take a while, since there multiple parties involved.